lib/widgets/game_view.dart

```diff
@@ -1,11 +1,20 @@
 import 'package:flutter/material.dart';
+import 'package:tic_tac_toe/widgets/rules_buttons.dart';
 
 import '../grids_page.dart';
 import '../model/boards.dart';
 import '../model/game.dart';
 import '../styles/colors.dart';
 import 'board.dart';
 
+var winner = '';
+
```

Comment:
Can you review code and remove everything that relates to winner detection?

Reply:
Done